libafl/src/feedbacks/mod.rs

```diff
@@ -113,6 +118,16 @@ where
         ret
     }
 
+    #[cfg(feature = "track_hit_feedbacks")]
+    fn last_result(&self) -> Option<bool>;
```

Comment:
CUT MY LIFE INTO PIECES
THIS IS MY LAST RESULT

Reply:
This track_hits

Reply:
lol I'm gonna change this to prev_result

Reply:
Nooo :D

Reply:
Actually, we do use `last_` a lot throughout the codebase, so I'd change it back, maybe add a papa roach reference into the comments